src/common/routes.tsx

```diff
@@ -93,6 +93,17 @@ export const routes = (
       </Route>
       <Route path="/vendors">
         <Route path="" element={<Vendors.Vendors />} />
+        <Route path="create" element={<Vendors.Create />} />
+
+        <Route path=":id">
```

Comment:
This should have the parent component `<Products.Product>`. Please read GitHub's comment first.